edx_exams/apps/lti/signals/handlers.py

```diff
@@ -1,13 +1,46 @@
 """
 edX Exams Signal Handlers
 """
+import logging
+
 from django.dispatch import receiver
 from lti_consumer.signals.signals import LTI_1P3_PROCTORING_ASSESSMENT_STARTED
 
+from edx_exams.apps.core.api import get_attempt_for_user_with_attempt_number_and_resource_id, update_attempt_status
+from edx_exams.apps.core.exceptions import ExamIllegalStatusTransition
+from edx_exams.apps.core.statuses import ExamAttemptStatus
+
+log = logging.getLogger(__name__)
+
 
 @receiver(LTI_1P3_PROCTORING_ASSESSMENT_STARTED)
 def assessment_started(sender, **kwargs):  # pylint: disable=unused-argument
     """
     Signal handler for the lti_consumer LTI_1P3_PROCTORING_ASSESSMENT_STARTED signal.
     """
-    print(f"LTI_1P3_PROCTORING_ASSESSMENT_STARTED signal received with kwargs: {kwargs}")
+    user_id = kwargs.get('user_id')
```

Comment:
since the thing that sends this is in another codebase I'm wondering if we should more explicitly check these args are correctly provided? I think the log under `if not attempt:` technically covers this but could leave us open to a future error if we're expecting these not to be null in an underlying function of `get_attempt_for_user_with_attempt_number_and_resource_id`

Reply:
Yeah, I was thinking about this and went back and forth a bit. I figured we own the LTI library so it would be pretty safe, but it doesn't hurt to be extra careful and not assume anything about the signal, so I'll check them here.

Reply:
@Zacharis278 I added some checking below. What do you think? Let me know if you'd rather I explicitly check for `None`.

Reply:
looks good!